# Returned message types
float32[] x
float32[] y
float32[] z
string[] orientation
int8[] num_dots_half1
int8[] num_dots_half2
